backend/src/routes/auth.ts

```diff
@@ -160,4 +165,86 @@ export const authRoute = new Elysia({ prefix: '/auth' })
             status: 'success',
             message: `${user.email} has been verified`,
         };
+    })
+    .post('/sign-in', async ({ request, cookie }) => {
+        const formData = await request.formData();
+
+        const validEmaillPass = signInFormSchema.safeParse({
+            email: formData.get('email'),
+            password: formData.get('password'),
+        });
+
+        if (!validEmaillPass.success) {
+            return {
+                status: '400',
```

Comment:
The API Reponse schema is status can be 'success' or 'error' not by the code. Consider using Reponse object to return status code and body instead